### ISSUE TITLE
Walkthrough disappears after window refresh

### ISSUE THREAD

**Issue by abitrolly** [openstreetmap/iD#7889]
1. Open Walkthrough in Firefox
2. Press F5

Expected - stay on the same page of the tutorial.
Actual - the tutorial is gone.

**Comment by quincylvania** [comment]
Hi and welcome, @abitrolly. This is an interesting idea. Is there a particular reason you need to refresh the page in the middle of things?

**Comment by abitrolly** [comment]
@quincylvania yep. For some reason the window I needed to click became positioned outside of clickable area and tutorial interface locked up.
